pkg/storaged/pages.jsx

```diff
@@ -383,16 +385,17 @@ const ActionButtons = ({ card }) => {
     const buttons = [];
     const items = [];
 
-    if (!card.actions)
+    if (!actions)
         return null;
 
-    for (const a of card.actions) {
+    for (const a of actions) {
         if (for_menu(a))
             items.push(make_menu_item(a));
         else
             buttons.push(
                 <StorageButton key={a.title} onClick={() => a.action(false)}
-                               kind={a.danger ? "danger" : null} excuse={a.excuse}>
+                               kind={a.danger ? "danger" : null} excuse={a.excuse}
```

Comment:
This added line is not executed by any test.  [Details](https://logs-cockpit.apps.ocp.cloud.ci.centos.org/pull-21927-b100443d-20250808-061643-fedora-42-devel/Coverage/lcov/github-pr.diff.gcov.html)